.storybook/preview.js

```diff
@@ -36,6 +36,7 @@ export const parameters = {
         'AlternativeVerticals',
         'SpellCheck',
         'ResultsCount',
+        'Geolocation',
         'LocationBias',
```

Comment:
will we mark LocationBias as deprecated in this pr?

Reply:
oh I didn't pick up that item, it's gonna be in another pr

Reply:
oh I see gotcha didn't realize we had another item